identity-example/src/test/java/com/stripe/android/identity/example/ExampleUnitTest.kt

```diff
@@ -0,0 +1,16 @@
+package com.stripe.android.identity.example
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
Do we need these dummy unit tests? Are they just to make use CI runs them?

Reply:
It's created automatically when the new module is created. Will clean them up in a follow up PR when we actually add new tests